core-bundle/contao/dca/tl_files.php

```diff
@@ -749,7 +749,7 @@ public function cutFile($row, $href, $label, $title, $icon, $attributes)
 	 */
 	public function dragFile($row, $href, $label, $title, $icon, $attributes)
 	{
-		return System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_RENAME_FILE) ? '<button type="button" title="' . StringUtil::specialchars($title) . '" ' . $attributes . '>' . Image::getHtml($icon, $label) . '</button> ' : ' ';
+		return System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_RENAME_FILE) ? '<button type="button" title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . Image::getHtml($icon) . '</button> ' : ' ';
```

Comment:
```suggestion
		return System::getContainer()->get('security.helper')->isGranted(ContaoCorePermissions::USER_CAN_RENAME_FILE) ? '<button type="button"' . $attributes . '>' . Image::getHtml($icon, $title) . '</button> ' : ' ';
```

Reply:
Are you sure that this works? So far, I have only changed `<a><img>` but not `<button><img>` because I was not sure if that would work.

Reply:
Note to myself: If we change this, we also have to add `'button img[alt]': {x: -9, y: 30},` in the tooltips controller.

Reply:
It seems you are right: https://stackoverflow.com/a/21641352